vkdgen.py

```diff
@@ -47,47 +47,53 @@
 alias int32_t = int;
 alias int64_t = long;
 
-@nogc pure nothrow {
-	uint VK_MAKE_VERSION(uint major, uint minor, uint patch) {
+@nogc pure nothrow {{

```

Comment:
Yes you are right, I realized this when I started the formatting. But firstly I think its is the right/modern way and it can always be checked when evaluating the result. The one time it bid me was when I checked the wrong version. Learned from that mistake.

> Also, please separate your commits. How the templates are parsed does not affect the changes in your commit message.

I don't understand this.